examples/bss_example.py

```diff
@@ -40,14 +54,15 @@
 if __name__ == '__main__':
 
     import argparse
-    parser = argparse.ArgumentParser(description='Demonstration of blind source separation using IVA.')
+    parser = argparse.ArgumentParser(description='Demonstration of blind source separation using IVA or ILRMA.')
     parser.add_argument('-b', '--block', type=int, default=2048,
             help='STFT block size')
+    parser.add_argument('-a', '--algo', type=str, default='ilrma',
```

Comment:
Here you can use the `choices` keyword that takes a list as argument to restrict the value of the argument to a few pre-defined values, e.g. `choices=['auxiva', 'ilrma']`. This will prevent an error below if the user provides a non-existent method.